src/main/java/kavitha/BagComparison/ArrayBag.java

```diff
@@ -0,0 +1,38 @@
+package kavitha.BagComparison;
+
+public class ArrayBag<T> {
```

Comment:
You wrote this class ? Or, it is provided by Tutor ? Can't we use the Bag<T> class which we implemented in earlier program ( the first coding exercise) ?

Reply:
I wrote this class for this particular question .Sure we can use the previous Bag class.